### PR TITLE
Convert the test-results.txt to html using a shell script

### PR DESCRIPTION
**Pull request by AnnMillikin**
Branch:  ann-11-reporting

Purpose:  to convert the test-results.txt to html using a shell script

In this branch, I created a reporting.sh script that converts the test-results.txt to test-results.html.  It's not especially pretty but it's a start and much improved of a text file.

- To test this branch, point your Jenkins pipeline to my branch.

branch to build ann-11-reporting

- Build with Parameters (accept the default parameters)
- Verify there are no failures
- Confirm the following artifacts exist

jenkins build artifactions for ann-11-reporting

- click the test-results.html
- confirm that the html page displays the results in a tabular format

- You could create a test failure by modifying the code and repeat the steps to confirm that a failure appears with FAIL in the table.

### CODE REVIEW

**Comment by KellyRyanGlobal** [comment]
> Branch: ann-11-reporting
> 
> Purpose: to convert the test-results.txt to html using a shell script
> 
> In this branch, I created a reporting.sh script that converts the test-results.txt to test-results.html. It's not especially pretty but it's a start and much improved of a text file.
> 
> * To test this branch, point your Jenkins pipeline to my branch.
> 
> branch to build ann-11-reporting
> 
> * Build with Parameters (accept the default parameters)
> * Verify there are no failures
> * Confirm the following artifacts exist
> 
> jenkins build artifactions for ann-11-reporting
> 
> * click the test-results.html
> * confirm that the html page displays the results in a tabular format
> * You could create a test failure by modifying the code and repeat the steps to confirm that a failure appears with FAIL in the table.

Branch failed the Ubuntu pipeline with the following errors
image

File: src/test_client.cc [hunk 1]
```diff
@@ -37,6 +37,10 @@ static void add_test() {
     LOG_INFO(TEST,  a << " + " << b << " = " << result);
 }
 
+TEST(functional, HandlesPositiveInput) {
```
Comment (KellyRyanGlobal) [inline src/test_client.cc:40]: What is this doing?

**Review by KellyRyanGlobal — COMMENTED** [review]
your build errors out but somehow it is executing the executable 
image

File: src/test_client.cc (hunk 1 above)
Comment (AnnMillikin) [reply src/test_client.cc]: Good catch!  That was residual code when I was playing around with GoogleTest.  Removed.

File: .gitmodules [hunk 2]
```diff
@@ -0,0 +1,3 @@
+[submodule "googletest"]
```
Comment (KellyRyanGlobal) [inline .gitmodules]: What is this doing?

**Review by KellyRyanGlobal — APPROVED** [review]

**Pull request merged**